lstchain/tools/lstchain_create_dl3_file.py

```diff
@@ -1,6 +1,14 @@
 """
 Create DL3 FITS file from given data DL2 file,
 selection cuts and/or IRF FITS files
+
+Simple usage with argument aliases and default config file for selection cuts:
```

Comment:
Put this into the description of the tool so people can actually see this when typing `--help`

Reply:
Ok, I will do so.

Reply:
To have it here, you could actually also do `description = __doc__`